lib/screen/profile_screen.dart

```diff
@@ -22,6 +25,7 @@ class _ProfileScreenState extends State<ProfileScreen>
     _tabController.addListener(() {
       setState(() {});
     });
+    _userModel = widget.userModel;
```

Comment:
No need to have another member variable here, using ``widget.userModel`` in the code is OK